<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyPort</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EasyPort.qrc">
    <normaloff>:/icons/EasyPorticon.png</normaloff>:/icons/EasyPorticon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_makefile">
       <attribute name="title">
        <string>Makefile</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QPlainTextEdit" name="text_makefile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Courier 10 Pitch</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="tabStopWidth">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tool_make_save">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="tool_make_addconf">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="EasyPort.qrc">
                <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tool_make_replaceconf">
              <property name="text">
               <string>Replace</string>
              </property>
              <property name="icon">
               <iconset resource="EasyPort.qrc">
                <normaloff>:/icons/right.png</normaloff>:/icons/right.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="list_make_config">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_distinfo">
       <attribute name="title">
        <string>Distinfo</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="tool_dist_autogen">
            <property name="text">
             <string>Auto-Generate</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="text_distinfo">
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_plist">
       <attribute name="title">
        <string>pkg-plist</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="tool_plist_save">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tool_plist_newdir">
            <property name="text">
             <string>Add</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="text_pkgplist">
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_desc">
       <attribute name="title">
        <string>pkg-descr</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QToolButton" name="tool_desc_save">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="EasyPort.qrc">
              <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="text_pkgdesc"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEasyPorts">
    <property name="title">
     <string>EasyPorts</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuPorts_Tree">
    <property name="title">
     <string>Ports Tree</string>
    </property>
    <widget class="QMenu" name="menuUpdate">
     <property name="title">
      <string>Update</string>
     </property>
     <property name="icon">
      <iconset resource="EasyPort.qrc">
       <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
     </property>
     <addaction name="actionWith_portsnap"/>
     <addaction name="actionWith_svn"/>
     <addaction name="actionWith_git"/>
    </widget>
    <addaction name="menuUpdate"/>
    <addaction name="separator"/>
    <addaction name="actionPackage_Current_Port"/>
    <addaction name="actionSubmit_FreeBSD_Port"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>Information</string>
    </property>
    <addaction name="actionPorter_s_Handbook"/>
    <addaction name="actionFreshports"/>
    <addaction name="actionRedports"/>
   </widget>
   <addaction name="menuEasyPorts"/>
   <addaction name="menuPorts_Tree"/>
   <addaction name="menuInformation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="statusTip">
    <string>Create a new FreeBSD port</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewPort"/>
   <addaction name="separator"/>
   <addaction name="actionLoad_Port"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionWith_portsnap">
   <property name="text">
    <string>With portsnap</string>
   </property>
  </action>
  <action name="actionWith_svn">
   <property name="text">
    <string>With svn</string>
   </property>
  </action>
  <action name="actionWith_git">
   <property name="text">
    <string>With git</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/config.png</normaloff>:/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionPorter_s_Handbook">
   <property name="text">
    <string>Porter's Handbook</string>
   </property>
  </action>
  <action name="actionFreshports">
   <property name="text">
    <string>Freshports</string>
   </property>
  </action>
  <action name="actionPackage_Current_Port">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Package Current Port</string>
   </property>
  </action>
  <action name="actionSubmit_FreeBSD_Port">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/freebsd.png</normaloff>:/icons/freebsd.png</iconset>
   </property>
   <property name="text">
    <string>Submit FreeBSD Port</string>
   </property>
  </action>
  <action name="actionRedports">
   <property name="text">
    <string>Redports</string>
   </property>
  </action>
  <action name="actionNewPort">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>NewPort</string>
   </property>
   <property name="toolTip">
    <string>Create a new port</string>
   </property>
  </action>
  <action name="actionLoad_Port">
   <property name="icon">
    <iconset resource="EasyPort.qrc">
     <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Load Port</string>
   </property>
   <property name="toolTip">
    <string>Load a port</string>
   </property>
   <property name="statusTip">
    <string>Load an existing FreeBSD port</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="EasyPort.qrc"/>
 </resources>
 <connections/>
</ui>
